src/components/LTAIPaymentForm.tsx

```diff
@@ -426,7 +528,7 @@ export function LTAIPaymentForm({ usdAmount, onPaymentSuccess }: Readonly<LTAIPa
 							Processing Payment...
 						</>
 					) : (
-						`${account?.chain === "solana" ? "" : "2. "}Pay with LTAI`
+						`${account?.chain === "solana" ? "" : "2. "}Pay with ${symbol}`
 					)}
 				</Button>
 			</div>
```

Comment:
ok I'll just refactor this, we need to make a generic form component that will just receive the symbol, a discount %, onProcessPayment etc